Dockerfile

```diff
@@ -11,10 +17,11 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
-FROM fedora
+FROM scratch
 
 MAINTAINER Avesh Agarwal <avagarwa@redhat.com>
```

Comment:
should we add your / manifold name ?

Reply:
You can only specify one maintainer in a Dockerfile.